site/examples/richtext.snapshot.test.tsx

```diff
@@ -0,0 +1,225 @@
+/** @jsx jsx */
+
+import { assertOutput, buildTestHarness } from 'slate-test-utils'
+import RichTextEditor from './richtext'
+import { jsx } from '../test-utils'
+import { fireEvent } from '@testing-library/dom'
+
+it('snapshot = user triggers bold hotkey and types with a collapsed selection', async () => {
```

Comment:
This is an example of a snapshot test ran with Slate Test Utils. I'm not a huge fan of snapshot tests in general, but added it for living docs in case folks prefer them!